src/config/mangata-dev.js

```diff
@@ -37,6 +40,7 @@ const Config = {
     ss58: 42,
     assets,
     pools,
+    instructionWeight: { refTime: new BN(150000000), proofSize: new BN(0) },
```

Comment:
Please define the below variable values at the top of this file. These are important numbers for running the test. 

feePerSecond -> FEE_PER_SECOND
instructionWeight.refTime -> WEIGHT_REF_TIME
instructionWeight.proofSize -> WEIGHT_PROOF_SIZE

Reply:
Please apply the same changes to all files. We can consolidate all the code into one common file and re-use it.